index.js

```diff
@@ -121,22 +124,21 @@ Liftoff.prototype.buildEnvironment = function (opts) {
     }, this);
   }
 
-  // use rechoir to autoload any required modules
-  var autoloads;
-  if (configPath) {
-    autoloads = rechoir.prepare(this.extensions, configPath, cwd, true);
-    if (autoloads instanceof Error) {
-      autoloads = autoloads.failures;
-    }
-    if (Array.isArray(autoloads)) {
-      autoloads.forEach(function (attempt) {
-        if (attempt.error) {
-          this.emit('requireFail', attempt.moduleName, attempt.error);
-        } else {
-          this.emit('require', attempt.moduleName, attempt.module);
+  registerLoader(this.extensions, configPath, cwd, this);
+
+  var configFiles = {};
+  if (isPlainObject(this.configFiles)) {
+    var exts = this.extensions, notfound = { path: null }, eventEmitter = this;
```

Comment:
`var exts = this.extension` can be defined higher in the function and used with the first `registerLoader` call also.